packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -110,6 +110,16 @@ export class ChoiceGroupBase extends BaseComponent<IChoiceGroupProps, IChoiceGro
   }
 
   public focus() {
+    const { options } = this.props;
+    if (options) {
+      for (const option of options) {
+        const elementToFocus = document.getElementById(`${this._id}-${option.key}`);
+        if (elementToFocus && elementToFocus.getAttribute('data-is-focusable') === 'true') {
+          elementToFocus.focus(); // focus on checked or default focusable key
+          return;
+        }
```

Comment:
I'm wondering if instead of searching for a raw attribute that a keyFocused key should be stored somehow, but I haven't really thought of any methods that don't also search through the options to find those that are checked... hmm